src/bindings/python/src/openvino/_ov_api.py

```diff
@@ -43,7 +43,7 @@ def __getattr__(cls, name: str) -> Any:
 class Model(object, metaclass=ModelMeta):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         if not args and not kwargs:
-            self.__model = ModelBase()
+            raise ValueError("The following argument types are supported")
```

Comment:
Will this display a proper error, with actual information about supported argument types?